webapp/src/Cache.ts

```diff
@@ -0,0 +1,49 @@
+/* global globalThis */
+
+import { debug } from '@/utils/log';
+import { envVarNotFound } from '@/utils/util';
+import LyraConfig from './utils/config';
```

Comment:
Fixed! I think we need an eslint rule for this at some point, because VSCode doesn't automatically create imports like this so I expect this will be a common mistake.